### ISSUE TITLE
spamgourmet user name

### ISSUE THREAD

**Issue by vasile-gh** [spamgourmet/first#1]
Hi there,

Is your username related in any way to  the https://www.spamgourmet.com/ website?

thx

**Comment by mikedlr** [comment]
@spamgourmet if there's anyone listening, since you don't seem to be using this name would you be willing to give it up for the actual code related to the spamgourmet.com web site?  It would be appreciated. 

For now I'm using this site instead:

- **[github.com/spamgourmet-com](https://github.com/spamgourmet-com)**

**Comment by vasile-gh** [comment]
yeah, that would be nice. I missed this name by just a day or so... it would be nice to use it to store the spamgourmet website code.
